README.md

```diff
@@ -4,6 +4,32 @@ A collection of documents that describe best practices for Canonical web team.
 
 These can be also be viewed at https://canonical-webteam.github.io/practices/.
 
+## Run site locally
+
+This site is built using [Jekyll](https://jekyllrb.com/).
+
+To run this site locally, you'll need to first [install Ruby](https://www.ruby-lang.org/en/documentation/installation/) on your local machine.
+
+Once you have installed Ruby, you can then install [Bundler](https://bundler.io) by running;
+
+```
+gem install bundler
```

Comment:
To get the correct version for the Gemfile.lock we can change this to `gem install bundler -v 1.17.3`

Reply:
Updated, thanks.